src/cyrsasl_external.erl

```diff
@@ -0,0 +1,61 @@
+-module(cyrsasl_external).
```

Comment:
Header, what is while for, how works.
What's the difference between this and cyrsasl_plain.